Woof/Woof/View/OwnerCardView.swift

```diff
@@ -0,0 +1,71 @@
+import SwiftUI
+
+/// OwnerCardView is a SwiftUI view that displays the profile information in card view style of an Owner.
+struct OwnerCardView: View {
+    @ObservedObject var viewModel = OwnerCardViewModel()
+    @State private var isEditMode: Bool = false
+
+    var body: some View {
+        VStack {
+            HStack {
+                Image(systemName: "person.circle")
+                    .resizable()
+                    .frame(width: 40, height: 40)
+                    .foregroundColor(Color.App.purpleDark)
+
+                VStack(alignment: .leading, spacing: 5.0) {
+                    Group {
```

Comment:
It's better not to use numbers where it's possible and create a type-safe format for them, it also helps to follow the app style guidelines. Please review the all code below.